backend/pkg/serde/service_integration_test.go

```diff
@@ -1258,8 +1258,8 @@ func (s *SerdeIntegrationTestSuite) TestDeserializeRecord() {
 		obj, ok := (dr.Value.DeserializedPayload).(map[string]any)
 		require.Truef(ok, "parsed payload is not of type map[string]any")
 		assert.Equal("345", obj["id"])
-		assert.Len(obj["dec_val"], 1)
-		assert.Len(obj["color"], 3)
+		assert.Len(obj["decVal"], 1)
+		assert.Len(obj["color"], 4)
```

Comment:
These are adjusted to reflect existing behaviour and functionality as is in master branch.